frontend/src/features/properties/list/PropertyListView.tsx

```diff
@@ -64,6 +69,8 @@ const getServerQuery = (state: {
     maxLandArea: decimalOrUndefined(maxLotSize),
     page: pageIndex + 1,
     quantity: pageSize,
+    parcelId: decimalOrUndefined(parcelId!),
```

Comment:
in general I'd rather we handle possibly undefined values with explicit checks/exceptions rather then the ! character.